src/java/org/apache/cassandra/tools/StandaloneScrubber.java

```diff
@@ -112,56 +112,58 @@ public static void main(String args[])
             System.out.println(String.format("Pre-scrub sstables snapshotted into snapshot %s", snapshotName));
 
             // if old-style json manifest, snapshot it
-            if (cfs.directories.tryGetLeveledManifest() != null)
-            {
+            if (cfs.directories.tryGetLeveledManifest() != null) {
                 cfs.directories.snapshotLeveledManifest(snapshotName);
                 System.out.println(String.format("Leveled manifest snapshotted into snapshot %s", snapshotName));
             }
 
-            LeveledManifest manifest = null;
```

Comment:
Question: Why move this to a class level, looks like this will be instantiated every time in the loop still?